hera_mc/args_test.py

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2016 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""This is meant to hold utility scripts for handling parts and connections
+
+"""
```

Comment:
Can you explain what this file does a little better. It says it's supposed to hold "utility scripts" but I just see a bunch of arg parser stuff. I can see how that could be reused but you don't seem to be using it in any of the other changes on this branch.